### ISSUE TITLE
Thunderbird - Beep Error when opening any  message

### ISSUE THREAD

**Issue by punkajim** [nvaccess/nvda#10687]
Steps to reproduce:
Actual behavior:
Run Thunderbird
Press enter to open a message in the folder 
Error beepp. NVDA continues to funtion as expected.
Expected behavior:
No error beep
System configuration
portable
NVDA 2019.3beta2
Windows version:
Windows 10 64
Thunderbird
Version
8.4.1
Have you tried any other versions of NVDA?
19.2 with no issues
[nvdaThunderbirdError.log](https://github.com/nvaccess/nvda/files/4063396/nvdaThunderbirdError.log)

**Comment by Brian1Gaff** [comment]
Just as a note as you are probably using a beta, like myself. A lot of items 
in windows and software give error noises with no obvious detriment to 
performance. I wonder if there exist somewhere a list of benign errors we 
can ignore?

Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message -----

**Comment by Adriani90** [comment]
I cannot reproduce this with NVDA 2019.3 Beta 2. I tried with eSpeak, One core voices and also with IBM TTS.
I also tried on installed and on portable version.
Did you try to restart your PC and see if it occurs after that?

**Comment by Adriani90** [comment]
The log file is not very useful since it is not in debug mode. We cannot see which action triggers the error. Please go to NVDA settings, general settings and set the log to debug. After that try to reproduce the error, open the log viewer and save the file and upload it again here. Thanks.

**Comment by punkajim** [comment]
@Adriani9
[NVDAThunderbirdDebug.txt](https://github.com/nvaccess/nvda/files/4067395/NVDAThunderbirdDebug.txt)
0 
I had rebooted.
I have discovered that when I open the same messages that generated the error beep a second time, no error occurs.
I have crated a debug log where I opened one message for the first time in beta2 and got the error.

**Comment by Adriani90** [comment]
Could you please try following?
1. Open NVDA menu and go to "tools"
2. Rund the com registry fixing tool and complete the process.

Is the error still reproducible after that?

**Comment by Adriani90** [comment]
For reference, this seems to be the relevant part of the log:

```
DEBUGWARNING - NVDAObjects.IAccessible.IAccessible._get_IA2Attributes (20:22:23.204) - MainThread (1932):
IAccessibleObject.attributes COMError (-2147220995, 'Object is not connected to server', (None, None, None, 0, None))
```

```
ERROR - speech.manager.SpeechManager._handleIndex (20:22:24.958) - MainThread (1932):
Error running speech callback
Traceback (most recent call last):
  File "speech\manager.pyc", line 375, in _handleIndex
  File "speech\commands.pyc", line 260, in run
  File "sayAllHandler.pyc", line 145, in <lambda>
  File "sayAllHandler.pyc", line 174, in lineReached
  File "documentBase.pyc", line 24, in makeTextInfo
  File "virtualBuffers\__init__.pyc", line 207, in __init__
  File "textInfos\offsets.pyc", line 460, in __init__
  File "virtualBuffers\__init__.pyc", line 225, in _getStoryLength
OSError: [WinError 1775] A null context handle was passed from the client to the host during a remote procedure call
```

**Comment by punkajim** [comment]
@

**Comment by punkajim** [comment]
@Adriani90 
I have done this successfully, and rebooted.
Still the same.
Does not happen on messages that I have already opened in this session and that are locally cached.
I use Gmail imap.
Error seems to occur whilst downloading new message from server.
